source/copilot-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _vsce-copilot-docs:
+
+==============
+/docs Command
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The ``/docs`` command provides MongoDB specific information to users,
+supplemented by links to MongoDB Documentation. The |copilot| is
+knowledgeable of the latest MongoDB Documentation and uses the
+MongoDB Docs Chatbot API to link users to information in addition to
```

Comment:
[Nit]

Similar to my previous comment, the use of "users" here feels a little strange to me, since our docs are typically written in the second person. Maybe we can drop "users" here as well?

Reply:
Done!